amundsen_application/static/css/_icons.scss

```diff
@@ -86,6 +86,11 @@ img.icon {
     background-color: transparent;
   }
 
+  &.icon-druid {
+    content: url('/static/images/icons/logo-druid.svg');
+    background-color: transparent;
+  }
```

Comment:
@ttannis Just wondering why some icons use `content` and others user `mask-image`.

Reply:
I can't remember the exact reason but our current logos are mix are logos that were generated by our previous contract designers in Sketch, and others that we found freely on the web. I recall some of the ones generated from Sketch worked differently for some reason and I think that's where `mask-image` came in.